dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/OrderAndPaginationExporterTest.java

```diff
@@ -1103,6 +1103,28 @@ void shouldOrderEventsByDataElementAndNotFilterOutEventsWithoutThatDataElement()
     assertEquals(List.of("pTzf9KYMk72", "D9PbzJY8bJM"), events);
   }
 
+  @Test
+  void shouldOrderEventsByDataElementAndFilterOutEventsWithoutThatDataElement()
```

Comment:
This test shows up as new but looks a lot like https://github.com/dhis2/dhis2-core/blob/5e446f580c5c99087ce0b753e58e5aa0b3ed44fa/dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/OrderAndPaginationExporterTest.java#L1003C8-L1023 :thinking:

Reply:
Yes, removing it.